config/clients/python/template/test/credentials_test.py.mustache

```diff
@@ -180,10 +180,3 @@ class TestCredentialsIssuer(unittest.TestCase):
         self.configuration.api_issuer = "https://issuer.fga.example:8080 "
         result = self.credentials._parse_issuer(self.configuration.api_issuer)
         self.assertEqual(result, "https://issuer.fga.example:8080/oauth/token")
-
-    # this should raise error
-    def test_invalid_issuer_bad_hostname(self):
```

Comment:
I think with this we'd be testing the language parsing for URLs so I propose we remove the test